020-LinuxFundamentals/Student/Challenge-09.md

```diff
@@ -0,0 +1,47 @@
+# Challenge 09 - Disks, partitions and file Systems
+
+[< Previous Challenge](./Challenge-08.md) - **[Home](../README.md)** - [Next Challenge >](./Challenge-10.md)
+
+## Pre-requisites 
+
+- Create a data disk woth the size of 5GB
+- Add the disk the the virtual machine 
+
+## Description
+
+This challenge you will be working with disks and partitions. Will be an opportunity to learn about linx filesystems and commands such as `fdisk`, `mkfs` and `mount`.
+
+- Identify the new disk added to the machine
+- Edit the disk partition table:
+    - Add a new partition with 500MB
+    - List and identify in the operating system the partition created
+    - Erase partition
+    - Check in operating system that the partition has been removed
```

Comment:
add "the" after "in"

Reply:
Still needs to be corrected to "Check in the operating system that the partition has been removed"